quipucords/api/scanjob/model.py

```diff
@@ -56,13 +56,15 @@ def __str__(self):
         """Convert to string."""
         return '{' + 'id:{}, '\
```

Comment:
You don't have to, but one way to protect yourself from typos when you have this many strings to substitute is to pass them as keyword arguments to format, so it would look like `'id:{id}, scan_type:{scan_type}, ...'.format(id=self.id, scan_type=self.scan_type, ...)`

Reply:
Good point.  I think most of the str method don't do this.  This may be a good fit and finish for all str methods.